nncf/tensorflow/__init__.py

```diff
@@ -19,11 +19,17 @@
 
 tensorflow_version = parse_version(tensorflow.__version__).base_version
 if not tensorflow_version.startswith(BKC_TF_VERSION[:-2]):
-    raise RuntimeError(
-         'NNCF only supports tensorflow=={bkc}, while current tensorflow version is {curr}'.format(
+    import warnings
+    warnings.warn("NNCF provides best results with tensorflow=={bkc}, "
+                   "while current tensorflow version is {curr} - consider switching to tensorflow=={bkc}".format(
          bkc=BKC_TF_VERSION,
          curr=tensorflow.__version__
     ))
+elif not ('2.4.3' <= tensorflow_version <= '2.5.2'):
```

Comment:
I guess we will support `tensorflow >=2.4.0, ==2.5.*`

Reply:
Sure